pages/desktop/user.py

```diff
@@ -165,8 +165,106 @@ def clear_field(self):
 
 class MyCollections(Base):
 
-    _header_locator = (By.CSS_SELECTOR, ".primary > header > h2")
+    _header_locator = (By.CSS_SELECTOR, '.primary > header > h2')
+    _my_favorites_locator = (By.CSS_SELECTOR, '.other-categories ul:nth-child(3) li:nth-child(3)')
 
     @property
     def my_collections_header_text(self):
         return self.selenium.find_element(*self._header_locator).text
+
+    def click_my_favorites(self):
+        self.selenium.find_element(*self._my_favorites_locator).click()
+        return MyFavorites(self.base_url, self.selenium)
+
+
+class MyFavorites(Base):
+
+    _page_title = 'My Favorite Add-ons :: Collections :: Add-ons for Firefox'
+    _header_locator = (By.CSS_SELECTOR, "h2.collection > span")
+
+    _edit_collection_locator = (By.CLASS_NAME, 'edit')
+    _favorites_add_ons_elements_locator = (By.CSS_SELECTOR, '.separated-listing .item')
+    _add_on_name_locator = (By.CSS_SELECTOR, '.item h3 a')
+    _add_to_favorites_locator = (By.CSS_SELECTOR, 'a.favorite')
+
+    def __init__(self, base_url, selenium, user_name=None):
```

Comment:
I don't think we need to overload the `__init__`. As far as I can tell we're not using the `user_name` kwarg anywhere..